src/pointer-analysis/value_set.h

```diff
@@ -242,28 +242,28 @@ class value_sett
     const namespacet &ns) const;
 
 protected:
```

Comment:
Prefer to make these `private` if at all possible.

Reply:
The virtual override points definitely need defer-to-base I'm afraid